Client/Frontend/Browser/GridTabViewController.swift

```diff
@@ -70,24 +73,88 @@ class GridTabViewController: UIViewController, TabTrayViewDelegate {
+    ) {
         self.tabManager = tabManager
         self.profile = profile
         self.delegate = tabTrayDelegate
         self.tabToFocus = tabToFocus
+        self.notificationCenter = notificationCenter
+
+        let contextualViewModel = ContextualHintViewModel(forHintType: .inactiveTabs,
+                                                          with: profile)
+        self.contextualHintViewController = ContextualHintViewController(with: contextualViewModel)
+        
         super.init(nibName: nil, bundle: nil)
-        collectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
-        collectionView.register(TabCell.self, forCellWithReuseIdentifier: TabCell.cellIdentifier)
-        collectionView.register(GroupedTabCell.self, forCellWithReuseIdentifier: GroupedTabCell.Identifier)
-        collectionView.register(InactiveTabCell.self, forCellWithReuseIdentifier: InactiveTabCell.Identifier)
-        collectionView.register(ASHeaderView.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: GridTabViewController.independentTabsHeaderIdentifier)
-        tabDisplayManager = TabDisplayManager(collectionView: self.collectionView, tabManager: self.tabManager, tabDisplayer: self, reuseID: TabCell.cellIdentifier, tabDisplayType: .TabGrid, profile: profile)
+        collectionViewSetup()
+    }
+
+    private func collectionViewSetup() {
+        collectionView = UICollectionView(frame: .zero,
+                                          collectionViewLayout: UICollectionViewFlowLayout())
+        collectionView.register(cellType: TabCell.self)
+        collectionView.register(cellType: GroupedTabCell.self)
+        collectionView.register(cellType: InactiveTabCell.self)
+        collectionView.register(
+            ASHeaderView.self,
+            forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader,
+            withReuseIdentifier: GridTabViewController.independentTabsHeaderIdentifier)
+        tabDisplayManager = TabDisplayManager(collectionView: self.collectionView,
+                                              tabManager: self.tabManager,
+                                              tabDisplayer: self,
+                                              reuseID: TabCell.cellIdentifier,
+                                              tabDisplayType: .TabGrid,
+                                              profile: profile,
+                                              cfrDelegate: self)
         collectionView.dataSource = tabDisplayManager
         collectionView.delegate = tabLayoutDelegate
 
         tabDisplayManager.tabDisplayCompletionDelegate = self
     }
 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tabManager.addDelegate(self)
+        view.accessibilityLabel = .TabTrayViewAccessibilityLabel
+
+        webViewContainerBackdrop = UIView()
```

Comment:
`webViewContainerBackdrop = UIView()` could be defined on the constant definition instead of having a force `!` ?